builder/frameworks/espidf.py

```
@@ -2106,45 +2022,47 @@ def _skip_prj_source_files(node):
 
 if ("arduino" in env.subst("$PIOFRAMEWORK")) and ("espidf" not in env.subst("$PIOFRAMEWORK")):
     def idf_lib_copy(source, target, env):
```

Ruff (0.12.2)

Unused function argument: `source`

(ARG001)